### PR TITLE
Add integration tests for Protobuf support

### PR DESCRIPTION
**Pull request by blacktooth**
*Description of changes:*
Adds Kafka and Kinesis integration tests for Protobuf support.

Tests successful.

### CODE REVIEW

File: integration-tests/src/test/java/com/amazonaws/services/schemaregistry/integrationtests/kafka/GlueSchemaRegistryKafkaIntegrationTest.java [hunk 1]
```diff
@@ -194,12 +209,15 @@ public void testProduceConsumeWithSchemaRegistry(final DataFormat dataFormat,
...
                 kafkaHelper.doProduceRecords(producerProperties, records);
 
-        ConsumerProperties consumerProperties = ConsumerProperties.builder()
-                .topicName(topic)
-                .avroRecordType(recordType.name()) // Only required for the case of AVRO
-                .build();
+        ConsumerProperties.ConsumerPropertiesBuilder consumerPropertiesBuilder = ConsumerProperties.builder().topicName(topic);
+        String recordType = getRecordType(dataFormat, avroRecordType);
+        if (dataFormat.equals(DataFormat.PROTOBUF)) {
+            consumerPropertiesBuilder.protobufMessageType(recordType);
```
Comment (hhkkxxx133) [inline integration-tests/src/test/java/com/amazonaws/services/schemaregistry/integrationtests/kafka/GlueSchemaRegistryKafkaIntegrationTest.java:215]: Why can't we directly set the `protobufMessageType` to `DYNAMIC_MESSAGE` here? `getRecordType` is using the same the conditional check.

File: integration-tests/src/test/java/com/amazonaws/services/schemaregistry/integrationtests/generators/ProtobufGenericBackwardDataGenerator.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            DynamicMessage.newBuilder(v2SchemaFileDescriptor.findMessageTypeByName(messageName))
+                .mergeFrom(
+                    baseMessage
+                        .toBuilder()
+                        .setStringType(BLOCKLISTED)
+                        .build()
+                        .toByteArray()
+                )
+                .build();
+        DynamicMessage v2DynamicMessage3 =
```
[the remark this answers is not in the document]
Comment (blacktooth) [reply integration-tests/src/test/java/com/amazonaws/services/schemaregistry/integrationtests/generators/ProtobufGenericBackwardDataGenerator.java:75]: Just added an extra message for fairness of test.

**Comment by blacktooth** [comment]
Addressed the comments.

**Review by hhkkxxx133 — APPROVED** [review]
LGTM

**Pull request merged**